static/app/utils/theme/theme.tsx

```diff
@@ -1172,7 +1172,9 @@ export const darkTheme = {
   },
 } satisfies SentryTheme;
 
-// Theme type exports
+/**
+ * @deprecated use import type {Theme} from '@emotion/react'
+ */
 export type SentryTheme = typeof lightTheme;
```

Comment:
can we just replace all of them?

Reply:
We dont seem to have any right now

Reply:
Let me remove it and see if anything breaks